.github/workflows/ci_tests_dev.yaml

```diff
@@ -139,13 +139,15 @@ jobs:
             -DGMT_USE_THREADS=TRUE
           cmake --build .
           cmake --build . --target install
+          cd ..
+          rm -rf gmt/
```

Comment:
Shouldn't be necessary to delete the gmt/ directory anymore right since we install to a temporary directory? Or is there something different on Windows?

Reply:
it's not necessary but deleting it does no harm.